sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -217,6 +218,9 @@ def test_gbk_side_input(self):
           main | beam.Map(lambda a, b: (a, b), beam.pvalue.AsDict(side)),
           equal_to([(None, {'a': [1]})]))
 
+  @unittest.skipIf(sys.version_info >= (3, 6, 0) and
+                   os.environ.get('RUN_SKIPPED_PY36_TESTS') != '1',
```

Comment:
It is sufficient to have one variable RUN_SKIPPED_PY3_TESTS - we only need a way to override the skip decorator.

Reply:
Yes, you're right. Updated, thanks!